README.md

```diff
@@ -1 +1,73 @@
+
+2. On a Second tab `Aws Settings` you will see. Please fill all reqiured fields.
+
+![UI second screen](assets/second-screen.png)
+
+| Field                                 | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| ------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Aws Access Key (Required)             | Your AWS access key ID.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| Aws Secret Key (Required)             | Your AWS secret key ID.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| Bucket Name (Required)                | The bucket name to which the PUT action was initiated. When using this action with an access point, you must direct requests to the access point hostname. The access point hostname takes the form AccessPointName-AccountId.s3-accesspoint.Region.amazonaws.com. When using this action with an access point through the Amazon Web Services SDKs, you provide the access point ARN in place of the bucket name. For more information about access point ARNs, see Using access points in the Amazon S3 User Guide. |
```

Comment:
```suggestion
| Bucket Name (Required)                | The bucket name to which the PUT action is initiated. When using this action with an access point, you must direct requests to the access point hostname. The access point hostname takes the form AccessPointName-AccountId.s3-accesspoint.Region.amazonaws.com. When using this action with an access point through the Amazon Web Services SDKs, you need to provide the access point ARN instead of the bucket name. For more information about access point ARNs, see Using access points in the Amazon S3 User Guide. |
```